sections/futures/MobileTrade/MobileDepositModal.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const MobileDepositModal = () => {
+	return <div />;
+};
```

Comment:
Is this a placeholder or something?

Reply:
Yeah. Basically forgot to implement the deposit/withdraw modals for mobile (since there aren't any designs for it).